Cart Flow/Custom Views/Buttons/CFPlusButton.swift

```diff
@@ -0,0 +1,43 @@
+//
```

Comment:
Your `CFPlusButton` class is almost identical to your `CFMinusButton` class and your `CFEditButton` class. There's an opportunity to simplify your codebase here with a quick refactor. Things totally work as-is, but if you moved some of the common code between these classes to a parent class that they all subclassed, you wouldn't have to change code in multiple places if you ever decided to, for instance, change the color of the icons in your app. Generally speaking, you want to follow a principle called DRY (Don't Repeat Yourself). In this case, the code breaks that principle since you've got the same code in multiple classes.

I'd suggest creating a `CFIconButton` class to encapsulate almost all of the logic of the `CFMinusButton`, `CFEditButton`, and `CFPlusButton` classes. Then have your `CFEditButton`, etc. classes inherit from `CFIconButton` instead of `UIButton`. More on inheritance in Swift here: https://docs.swift.org/swift-book/LanguageGuide/Inheritance.html.

PS - I highly recommend reading "The Pragmatic Programmer" if you haven't already. A quick overview of the suggestions in that book (including DRY) can be found here: https://cheatography.com/marconlsantos/cheat-sheets/pragmatic-programming/

Reply:
Ah good call! One of my biggest missteps/learnings on this project was not being thorough enough in the prototyping stage. Ended up not using some buttons that I thought I would, and also changing layouts after I had already built them.